### PR TITLE
i3180 - pdf print ext

### LINKED ISSUE #3180: Additional PDF pages
# Story

As an Admissions site owner, I want to convert additional pages to a PDF so that these pages can print consistently between browsers.

# Additional Info

There are some additional pages that are not areas of study that also need to be converted to PDFs to print. These should be listed at the bottom of the list of print links. These will be more than two pages.

The additional pages can be seen at the bottom of this page. https://admissions.uiowa.edu/at-iowa-print

Here are the existing pages on the new site. I don't believe the Global Assessment Certificate Program (GAC) page has been created yet on the new site.
https://admissions.prod.drupal.uiowa.edu/academics/ap
https://admissions.prod.drupal.uiowa.edu/academics/college-level-examination-program-clep
https://admissions.prod.drupal.uiowa.edu/academics/ib
https://admissions.prod.drupal.uiowa.edu/academics/cambridge-international-exams-cie

# Proposed solution

### PR DESCRIPTION
**Pull request by dnkemper**
Closes #3180 

# How to test

checkout branch
blt ds --site=admissions.uiowa.edu
blt frontend
go to http://admissions.local.drupal.uiowa.edu/iowa-print
Click the first two links and see that they redirect you to a pdf version of webpage
For now, only content added to the body field will display on pdf of the page.

Also: Go to the content overview, download the page as a pdf and see that the body text is visible

### CODE REVIEW

File: config/admissions.uiowa.edu/config_split.config_split.admissions.yml [hunk 1]
```diff
@@ -20,13 +20,45 @@ graylist:
   - core.entity_form_display.node.transfer_tips.default
   - core.entity_form_display.paragraph.admissions_card.default
   - core.entity_form_display.paragraph.admissions_requirement.default
+  - core.entity_view_display.node.area_of_study.default
```
Comment (joewhitsitt) [inline config/admissions.uiowa.edu/config_split.config_split.admissions.yml]: This seems to undo the clean up that @pyrello did in https://github.com/uiowa/uiowa/commit/639c8f21e1c64399c494058930d7ce8eeb506abc#diff-0d6789e2b54025afe51969cefc90c6014041f181116fe88518838b424bc3219d using wildcards to cover these.

Comment (dnkemper) [reply config/admissions.uiowa.edu/config_split.config_split.admissions.yml]: @joewhitsitt you know it's a problem when I can't come up with an explanation for this... 😬

**Comment by joewhitsitt** [comment]
pulling down to test

**Comment by joewhitsitt** [comment]
Not sure if PDF quality is in-scope for this PR and/or if these are content problems, not style problems. Found on the two links recommended from the testing instructions.

Screen Shot 2021-03-16 at 10 08 47 AM
Screen Shot 2021-03-16 at 10 08 15 AM
Screen Shot 2021-03-16 at 10 08 39 AM

**Comment by bspeare** [comment]
I think we can remove that fixed footer from these PDFs.   These lines can go away: https://github.com/uiowa/uiowa/blob/8cdeefed7b2d5ae082642716416dfbd0770e6e55/docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/node--page--pdf.html.twig#L103-L113.

**Comment by dnkemper** [comment]
@bspeare thank you! Done!

**Comment by joewhitsitt** [comment]
re-checking

**Review by joewhitsitt — APPROVED** [review]
Functionally works as describes. Thanks for the cleanup.

**Comment by dnkemper** [comment]
THANK YOU, @joewhitsitt

**Pull request merged**